dev-lang/ghc/ghc-7.10.3.ebuild

```diff
@@ -182,7 +182,7 @@ update_SRC_URI() {
 		set -- $p
 		pn=$1 pv=$2
 
-		SRC_URI+=" mirror://hackage/package/${pn}/${pn}-${pv}.tar.gz"
+		SRC_URI+=" https://hackage.haskell.org/package/${pn}/${pn}-${pv}.tar.gz"
```

Comment:
This is not consistent to other changes. The rest of URL mangling in pull request does the equivalent of `${pn}-${pv}/${pn}-${pv}.tar.gz` and not `${pn}/${pn}-${pv}.tar.gz`. They should agree.

Reply:
Yeah, my grep+sed wnet for `packages` but this one had `package` instead. Will improve.